packages/code-du-travail-frontend/src/modules/home/__tests__/HomeSearch.test.tsx

```diff
@@ -52,6 +52,7 @@ describe("<HomeSearch />", () => {
     expect(getByText("congés payés et maladie")).toBeInTheDocument();
 
     userAction.click(congesSansSolde);
+    fireEvent.submit(byTestId("search-input").get());
```

Comment:
Plutôt qu'un testid, ca pourrait être cool de récupérer l'input par son label 
```suggestion
    fireEvent.submit(byLabelText("Recherchez par mots-clés").get());
```

Reply:
j'ai essayer mais ça marche pas. Je penses que c'est parce que le label est comme ça :
```       
  <label
      class="fr-label"
      for="input-:r1:"
   >
      Recherchez par mots-clés
      <span
        class="fr-hint-text"
      >
        par exemple : congés payés, durée de préavis
      </span>
    </label>
```

Reply:
ah oui en effet, peut être mettre en regex dans ce cas